docs/rules/no-hashtag-description.md

```diff
@@ -6,7 +6,7 @@
 - Requires GraphQL Operations: `false` [ℹ️](../../README.md#extended-linting-rules-with-siblings-operations)
 
 Requires to use `"""` or `"` for adding a GraphQL description instead of `#`.
-This rule allows you to use hashtag for comments, as long as it's not attached to a AST definition.
+Allows to use hashtag for comments, as long as it's not attached to an AST definition.
```

Comment:
remove redundant `This rule allows you`
![image](https://user-images.githubusercontent.com/7361780/125083331-8850a680-e0c8-11eb-8ba9-ca2c915f40b1.png)